src/EditorFeatures/CSharpTest/Diagnostics/AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest.cs

```diff
@@ -19,5 +19,50 @@ public abstract class AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest :
 
         protected override TestWorkspace CreateWorkspaceFromFile(string initialMarkup, TestParameters parameters)
             => TestWorkspace.CreateCSharp(initialMarkup, parameters.parseOptions, parameters.compilationOptions);
+
+#pragma warning disable CS0108 // Member hides inherited member; missing new keyword
```

Comment:
❔ Why not just put the `new` keyword, since the code is inside a `#if` anyway?

Reply:
I was being lazy :-) I will change.